crates/aptos/src/node/mod.rs

```diff
@@ -1009,8 +1009,14 @@ impl TryFrom<&ValidatorConfig> for ValidatorConfigSummary {
         Ok(ValidatorConfigSummary {
             consensus_public_key,
             // TODO: We should handle if some of these are not parsable
-            validator_network_addresses: config.validator_network_addresses()?,
-            fullnode_network_addresses: config.fullnode_network_addresses()?,
+            validator_network_addresses: match config.validator_network_addresses() {
+                Ok(addresses) => addresses,
+                Err(_) => vec![],
```

Comment:
Maybe use `eprintln!("")` to print out the error so that we know that it's not empty?

Either that or we probably need to fix the parsing of the network addresses to skip broken ones

Reply:
@gregnazario  Use `eprintln!("")` to print out the error on top of the whole result?

I think that works, another idea I have is to divide it into following:

Empty Address -> return vec![]
Not parsable -> Return error instead of actual result
Valid address -> Ok()

Reply:
That also works as well